scrapinghub/client/projects.py

```diff
@@ -121,4 +122,57 @@ def __init__(self, client, projectid):
         self.activity = Activity(_Activity, client, projectid)
         self.collections = Collections(_Collections, client, projectid)
         self.frontiers = Frontiers(_HSFrontier, client, projectid)
-        self.settings = Settings(client._hsclient, projectid)
+        self.settings = Settings(_Settings, client, projectid)
+
+
+class Settings(_Proxy):
+    """Class representing job metadata.
+
+    Not a public constructor: use :class:`Project` instance to get a
+    :class:`Settings` instance. See :attr:`Project.settings` attribute.
+
+    Usage::
+
+    - request project settings
+
+        >>> settings = project.settings
+        >>> settings
+        Settings({'default_job_units': 1, 'job_runtime_limit': 24})
+
+    - get length of project settings
+
+        >>> len(project.settings)
+        2
+
+    - iterate through project settings keys
+
+        >>> project.settings.iter()
+        <dict_keyiterator at 0x106ae9cc8>
+
+    - list project settings keys
+
+        >>> project.settings.list()
+        ['default_job_units', 'job_runtime_limit']
```

Comment:
actually not sure how much it makes sense to follow iter()/list() convention in case of Settings and Metadata because they are rather dict-like objects and there are expectations that they will behave as dicts (so keys(), iterkeys() should work instead of list(), iter())